src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/AvatarImage.xml

```diff
@@ -16,6 +16,8 @@
            xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
 >
     <resource class="%sylius.model.avatar_image.class%" shortName="AvatarImage">
+        <attribute name="route_prefix">/admin</attribute>
```

Comment:
Could be prefixed as well, I suppose, so we will have only one place to adjust it in case

Reply:
If we already have: 
```yaml
    sylius.security.new_api_route: "/new-api"
    sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
```
we could add another parameter
```yaml
    sylius.security.new_api_route: "/new-api"
    sylius.security.new_api_route.admin_prefix: "/admin"
    sylius.security.new_api_admin_route: "%sylius.security.new_api_route%%sylius.security.new_api_route.admin_prefix%"
```
And then all of this attributes could be "simplified" to:
```suggestion
        <attribute name="route_prefix">"%sylius.security.new_api_route.admin_prefix%"</attribute>
```
But I'm not sure if it is not overcomplicated solution

Reply:
I will leave as it is now, because there is a small problem with `/` when there is a value taken from parameter